geomstats/geometry/stiefel.py

```diff
@@ -63,22 +85,41 @@ def random_uniform(self, n_samples=1):
 
         If :math:`Z(p,n) \sim N(0,1)`, then :math:`St(n,p) \sim U`,
         according to Haar measure:
-        :math:`St(n,p) := Z(Z^TZ)^{-1/2}`
+        :math:`St(n,p) := Z(Z^TZ)^{-1/2}`.
+
+        Parameters
+        ----------
+        n_samples : int, optional
+            Number of samples.
+
+        Returns
+        -------
+        samples : array-like, shape=[n_samples, n, p]
+            Samples on the Stiefel manifold.
         """
         std_normal = gs.random.normal(size=(n_samples, self.n, self.p))
         std_normal_transpose = gs.transpose(std_normal, axes=(0, 2, 1))
         aux = gs.einsum('nij,njk->nik', std_normal_transpose, std_normal)
         sqrt_aux = gs.linalg.sqrtm(aux)
         inv_sqrt_aux = gs.linalg.inv(sqrt_aux)
-        point = gs.einsum('nij,njk->nik', std_normal, inv_sqrt_aux)
+        samples = gs.einsum('nij,njk->nik', std_normal, inv_sqrt_aux)
```

Comment:
Might as well do `return gs.einsum` ;)

Reply:
True x)! I wanted to have a name to put in the Returns part of the docstring. I'll leave it for now ^^